modules/javafx.base/src/main/java/javafx/util/converter/BaseStringConverter.java

```diff
@@ -25,32 +25,37 @@
 
 package javafx.util.converter;
 
-import java.time.format.DateTimeFormatter;
-import java.time.format.FormatStyle;
-import java.util.Locale;
-
-public class LocalDateStringConverterShim {
-
-    public static DateTimeFormatter getldtConverterFormatter(LocalDateStringConverter c) {
-        return c.ldtConverter.formatter;
-    }
-    //-----------------------------------------------------
-
-    //---------
-    public static Locale getldtConverterLocale(LocalDateStringConverter c) {
-        return c.ldtConverter.locale;
+import javafx.util.StringConverter;
+
+/// A base class containing common implementations for `StringConverter`s as noted in the @implNote of `StringConverter`.
```

Comment:
In light of possible backporting, it's been decided earlier not to use markdown comments.

Reply:
Yes, that was what I had suggested earlier. As I mentioned above I think this is one case where using markdown comments seems reasonable.